packages/tpetra/core/src/Tpetra_Details_computeOffsets.hpp

```diff
@@ -204,45 +204,20 @@ class ComputeOffsetsFromConstantCount {
   ComputeOffsetsFromConstantCount (const offsets_view_type& offsets,
                                    const count_type count) :
     offsets_ (offsets),
-    count_ (count),
-    size_ (offsets_.extent (0) == 0 ?
-           size_type (0) :
-           size_type (offsets_.extent (0) - 1))
+    count_ (count)
```

Comment:
We're in the middle of debugging / rewriting this code.  Would it be OK to hold off on changes here until we finish that?  It's related to #5179.  Thanks!

Reply:
I revert my commit but I think that this would simplify the code for the constant count case. After you are finished debugging please include this modification.

Reply:
@kyungjoo-kim I don't think you have to revert the commit -- you should just able to resolve the merge conflict through the interface -- but reverting is fine.  Also, I think debugging doesn't actually affect this function.